src/views/Search.vue

```diff
@@ -18,51 +18,138 @@
         </div>
       </header>
 
-      <div class="col-xs-12 col-md-9">
-        <b-pagination
-          :total-rows="numFound"
-          :per-page="rowsPerPage"
-          v-model="currentPage"
-          responsive="true"
-          class="table-sm table-border-soft mt-2"
-          size="md"
-        />
-        <div class="search-results-rows">
-
-          <div v-if="searchResults && searchResults.length > 0 ">
-            <h3><span class="searchTerm">{{ query }}</span>  has <span class="searchTerm">{{ numFound }}</span> matches</h3>
-            <b-table
-              ref="results-table"
-              :fields="fields"
-              :items="rowsProvider"
-              :current-page="currentPage"
-              :per-page="rowsPerPage"
-              striped
-              responsive="true"
-              class="table-sm table-border-soft test-search-results-table"
-              hover>
-              <template
-                slot="label"
-                slot-scope="row"
-              >
-                <router-link :to="row.item.toLink">
-                  {{ row.item.label }}
-                </router-link>
-              </template>
-              <template
-                slot="highlight"
-                slot-scope="row"
-              >
-                <span v-html="row.item.highlight"/>
-              </template>
-            </b-table>
+      <div class="row">
+        <div
+          class="container col-md-3"
+        >
+          <div
+            v-if="categoryFilters.length>0 || taxonFilters.length>0"
```

Comment:
Not sure why the ESLint isn't picking this up, but you should have spaces around binary operators like `>`.